packages/graphqlgen/src/generators/common.ts

```diff
@@ -41,6 +41,7 @@ export function fieldsFromModelDefinition(
   // If model is of type `type TypeName = { ... }`
   if (
     modelDef.kind === 'TypeAliasDefinition' &&
+    (modelDef as TypeAliasDefinition).getType() &&
```

Comment:
I know you are just adding to convention that was already there, but I don't believe that this `as` cast is necessary. The first part of the conditional should be adequate as a type guard, since modelDef is a discriminated union (kind acting as the discriminant).

Reply:
You're absolutely right, I can make that change. I tried to fix it up everywhere but that turned out to be tricky.